etc/vzlogger.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=vzlogger
-After=syslog.target network.target ntp.service
+After=syslog.target NetworkManager-wait-online.service ntp.service
+Wants=NetworkManager-wait-online.service
```

Comment:
> Don't order your own systemd services after NetworkManager-wait-online.service directly. Instead if necessary, order your services after network-online.target. Even better is to have your services react to network changes dynamically and don't order them with respect to network-online.target at all.
-- https://developer-old.gnome.org/NetworkManager/stable/nm-online.html

Reply:
also, this needlessly introduces a dependency on network-manager (there are other network-management daemons users might be using, or none at all).